config/debridge.json

```diff
@@ -0,0 +1,44 @@
+{
+  "flags": {
+    "UNWRAP_ETH": 0,
+    "REVERT_IF_EXTERNAL_FAIL": 1,
+    "PROXY_WITH_SENDER": 2,
+    "SEND_HASHED_DATA": 3,
+    "SEND_EXTERNAL_CALL_GAS_LIMIT": 4,
+    "MULTI_SEND": 5
+  },
+  "config": {
+    "hardhat": {
+      "chainId": 1,
```

Comment:
Is this correct that hardhat has same chainID as mainnet?
Usually hardhat has like 31337 or so